src/renderer/views/Channel/Channel.js

```diff
@@ -416,23 +416,23 @@ export default defineComponent({
       })
     },
 
-    channelInvidiousNextPage: function () {
+    channelInvidiousVideos: function (fetchMore) {
       const payload = {
         resource: 'channels/videos',
         id: this.id,
         params: {
           sort_by: this.videoSortBy,
-          page: this.latestVideosPage
```

Comment:
Please remove all other references to `latestVideosPage` as that variable is no longer used.